trade-finance-manager-ui/server/controllers/utilisation-reports/edit-payment/index.test.ts

```diff
@@ -444,7 +444,48 @@ describe('controllers/utilisation-reports/edit-payment', () => {
           body: aPostEditPaymentRequestBody(),
         });
 
-      it('redirects to /utilisation-reports/:reportId when the edit payments form is valid', async () => {
+      beforeEach(() => {
+        jest.mocked(api.editPayment).mockResolvedValue();
+      });
+
+      afterEach(() => {
+        jest.resetAllMocks();
+      });
+
+      it('edits the payment with the supplied data', async () => {
+        // Arrange
+        const { req, res } = getHttpMocks();
+
+        const user = aTfmSessionUser();
+        const userToken = 'abc123';
+        req.session.user = user;
+        req.session.userToken = userToken;
+
+        const paymentAmount = 100;
+        req.body.paymentAmount = '100';
+
+        const datePaymentReceived = new Date('2024-5-12');
+        req.body['paymentDate-day'] = '12';
+        req.body['paymentDate-month'] = '5';
+        req.body['paymentDate-year'] = '2024';
+
+        const paymentReference = 'A payment reference';
+        req.body.paymentReference = paymentReference;
+
+        const parsedFormValues: ParsedEditPaymentFormValues = {
+          paymentAmount,
+          datePaymentReceived,
+          paymentReference,
+        };
```

Comment:
I used to prefix things with `expected` and then stopped because I started to think it was redundant in many cases. In this case though I agree that `expected` makes sense to use